src/main/java/bio/terra/workspace/db/WorkspaceDao.java

```diff
@@ -168,7 +168,7 @@ public WorkspaceCloudContext getCloudContext(UUID workspaceId) {
   /** Update the cloud context of the workspace, replacing the previous cloud context. */
   @Transactional(propagation = Propagation.REQUIRED, isolation = Isolation.SERIALIZABLE)
   public void updateCloudContext(UUID workspaceId, WorkspaceCloudContext cloudContext) {
-    if (cloudContext.googleProjectId().isPresent()) {
+    if (cloudContext.googleProjectId() != null) {
```

Comment:
A CloudContext is not specific to GCP, so it's valid to have a CloudContext without a google project ID. It's a little weird while there's only one supported cloud, but in general I think that's fine.

I think the correct solution here is returning an Optional<String> from `googleProjectId()`, but per above comments that's not supported in AutoValue unless we make the field Optional<String> instead of String.